data/sql/updates/pending_db_world/rev_1607503600897995800.sql

```diff
@@ -0,0 +1,13 @@
darán en llegar.','18019'),('3103','esMX','Ah, por fin has venido. Sabía que encontrarías el camino hacia mí. La luz sagrada brilla sobre ti y el camino que has escogido. Estos tiempos son duros, la legión ardiente todavía tiene presencia en Azeroth, la totalidad de Kalimdor busca nuevas formas de defenderse de sus propias tribulaciones, y ahora está usted para ayudar a todos los que pueda.','18019'),('3100','esMX','Ah, tienes mi carta $N... Bien.$B$BHa habido una afluencia de guerreros en bosque de elwynn recientemente, lo cual es bueno para ventormenta, pero malo para los kobolds y defias en el área. ','18019'),('3104','esMX','Hola $N. Soy Khelden. ¿Hay algo que pueda hacer por ti?','18019'),('54','esMX','¿Tienes algo de McBride? Villanorte es un jardín comparado con bosque de elwynn, pero me pregunto qué tiene que informar el alguacil McBride.$B$BDéjame tener sus papeles... ','18019'),('47','esMX','¡Psst! ¿Tienes ese polvo de oro para mí...para mí?','18019'),('60','esMX','¿Ya reuniste las velas?','18019'),('61','esMX','¿Un envío de mi hermano? ¡Espléndido! ¡Hoy la fortuna me brilla verdaderamente!','18019'),('62','esMX','¿Qué tienes que informar $N? ¿Has estado en la mina Abisal?','18019'),('76','esMX','Hola $N. ¿Qué tiene que informar? ¿Has explorado la cantera de jaspe?','18019'),('11','esMX','Hola $N. ¿Has estado matando a los Gnolls?','18019'),('176','esMX','Sí, Hogger ha sido un verdadero dolor de cabeza para mí y mis hombres. ¿Tienes algo que informar sobre la bestia?','18019'),('52','esMX','¿Mataste a aquellos osos y lobos?','18019'),('83','esMX','Me estoy quedando sin lino. ¿Tienes algo para mí?','18019'),('5545','esMX','Esa fecha límite no se está alejando. Por favor apresúrate y recoge los fardos de madera.','18019'),('71','esMX','Hola $N. ¿Descubriste el destino de Rolf y Malakai?','18019'),('59','esMX','He sido comisionada por el ejército de ventormenta para proveer a su gente armaduras de tela y cuero.$B$BSi tienes material para mí, entonces estaría encantada de hacer algo por ti.','18019'),('46','esMX','¿Cómo va la cacería $N?','18019'),('106','esMX','¿Tienes qué? Maybell es la luz de mi aburrida vida. ¡Rápido, déjame ver su carta!','18019'),('107','esMX','Tienes algo de "Abuela" Pedregosa, ¿eh? ¡No he visto a Mildred en años! Me pregunto qué tiene que decir...','18019'),('114','esMX','¿Le entregaste mi carta a Tommy Joe? ¿Qué dijo?','18019'),('88','esMX','¿La has visto ya? ¿La has visto?','18019'),('86','esMX','No creo que esté bien alimentar al chico que robó mi collar en primer lugar, pero si eso es lo que se necesita para recuperar lo que es mío ¡entonces que así sea!$B$B¿Tienes carne de jabalí?','18019'),('84','esMX','¡Ugh, me muero de hambre! ¿Ya tienes ese pastel para mí, $N?','18019'),('87','esMX','Hola $N. ¿Has encontrado mi collar?','18019'),('16','esMX','Cultivar es un trabajo sediento, y siempre estoy buscando agua de mamantial refrescante.$B$BSi tienes agua, entonces estoy dispuesto a hacer un intercambio.','18019');
```

Comment:
1. Remove quotes from numbers. They are ints not strings.
2. Keep everything on new lines like this
DELETE XXX
INSERT XXX VALUES
(X, x, x),
(X, x, x);

Reply:
I did it, check it ;-)